GoodDollar/GoodServer
# Dont block user on mautic

# Description
during signup we block until data is updated in mautic/database etc this is not critical for user, so i've removed blocking

- dont block on user signup
- removed topwallet form signup (user now uses the faucet contract through app)
- removed temp mautic contact creation on verify email, no longer required since we send through SES

QA
- check user signup is working as expected
- new user doesnt have gas issue
- verify email step in signup(with passwordless) and when editing on profile works fine and also updates the user email on mautic correctly
